Kitodo/src/main/java/org/kitodo/production/helper/VariableReplacer.java

```diff
@@ -101,17 +101,17 @@ public String replace(String inString) {
         // replace paths and files
         try {
             // TIFF writer scripts will have a path without an end slash
-            String tifPath = replaceSlashAndSeparator(processService.getImagesTifDirectory(false, this.process.getId(),
+            final String tifPath = replaceSlashAndSeparator(processService.getImagesTifDirectory(false, this.process.getId(),
                 this.process.getTitle(), this.process.getProcessBaseUri()));
-            String imagePath = replaceSlashAndSeparator(fileService.getImagesDirectory(this.process));
-            String origPath = replaceSlashAndSeparator(processService.getImagesOriginDirectory(false, this.process));
-            String processPath = replaceSlashAndSeparator(processService.getProcessDataDirectory(this.process));
-            String metaFile = replaceSlash(fileService.getMetadataFilePath(this.process, false, false));
-            String ocrBasisPath = replaceSlashAndSeparator(fileService.getOcrDirectory(this.process));
-            String ocrPlaintextPath = replaceSlashAndSeparator(fileService.getTxtDirectory(this.process));
-            String sourcePath = replaceSlashAndSeparator(fileService.getSourceDirectory(this.process));
-            String importPath = replaceSlashAndSeparator(fileService.getImportDirectory(this.process));
-            String prefs = ConfigCore.getParameter(ParameterCore.DIR_RULESETS) + this.process.getRuleset().getFile();
+            final String imagePath = replaceSlashAndSeparator(fileService.getImagesDirectory(this.process));
```

Comment:
Don't we have naming convetions for constants? Capital letters only, with underscores between individual words?

Reply:
Actually it's not a constant

Reply:
Yes, you are right, apparently it's a `final variable`, and not a `constant`. And according to [Wikipedia](https://en.wikipedia.org/wiki/Final_(Java)), the naming convention I mentioned only applies to the later:
> Unlike the value of a constant, the value of a final variable is not necessarily known at compile time. It is considered good practice to represent final constants in all uppercase, using underscore to separate words.